Dependencies/xr/Include/XR.h

```diff
@@ -133,7 +146,10 @@ namespace xr
                 Frame(System::Session::Impl&);
                 ~Frame();
 
-                void GetHitTestResults(std::vector<Pose>&, Ray) const;
+                void GetHitTestResults(std::vector<HitResult>&, Ray) const;
+                Anchor CreateAnchor(Pose, void*) const;
```

Comment:
Seeing `void*` here is a bit confusing, particularly since there is no parameter name. Since this is basically an opaque pointer to a platform native type, maybe we should give this type a name, e.g.
```c++
using NativeTrackable = void; // -or-
using NativeTrackablePtr = void*; // -or-
typedef struct NativeTrackable_ NativeTrackable; // ARCore does something like this, but I don't really get what it's doing.
```

Reply:
Good idea the vanilla void* makes it pretty confusing in the context of xr.cpp.